package.json

```diff
@@ -49,6 +49,7 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
+    "apollo-server-express": "2.9.16",
```

Comment:
Good choice.  I notice the [GraphQL docs recommend this as one of the preferred options](https://graphql.org/learn/serving-over-http/#node).